content/1.7/back-office/shipping/carriers/Add-edit-carrier.md

```diff
@@ -0,0 +1,178 @@
+
+# Multistore (2nd step)
+
+**This step is displayed only when the multistore is enabled and if there are several shops.**
+
+The shop association block is displayed when the multistore is enabled and if several shops are configured.
+
+👉 See the [specs of the shop association block]({{<ref "../../multistoregeneralspecs.md#shop-association-block">}} "Shop association block") 
+
+### Call to action
+ 
+ - **Previous.** If there is an error, this button is disabled. After clicking on it, the previous step is displayed.
+ - **Next.** If there is an error, this button is disabled. After clicking on it, the next step is displayed.
+- **Finish.** If there is an error, this button is disabled. After clicking on it, the carrier is saved and the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page is displayed with the following success message: "Successful creation" if a carrier was added or "Successful update" if a carrier was edited.
+
+# Shipping locations and costs (3rd step)
+ 
+### Call to action
+
+A call to action button at the top right is displayed:
+
+ - Cancel:
+Pressing the button redirects to the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page.
+
+## Logo
+
+Firstly on the left, the logo is displayed as soon as it's uploaded. Once the logo has been uploaded, a trash icon appears on the right of the logo. When trying to delete the logo, a message is displayed: "Are you sure you want to delete the logo?". Pressing "Ok" will remove the uploaded logo, and the default logo will re-appear in the same place.
+
+## Steps
+
+The different steps are displayed:  
+
+ - 1 General settings
+ - 2 Multistore (If multistore is enabled)
+ - 3 Shipping locations and costs
+ - 4 Size, weight, and group access
+ - 5 Summary
+
+## Fields
+
+- *Add handling costs.** This toggle allows choosing if the handling costs are added to the final price or not. By default, this toggle is disabled.
+A tooltip is displayed: "Include the handling costs (as set in Shipping > Preferences) in the final carrier price."
+- **Free shipping.** This toggle allows choosing if the carrier is free or not. By default, this toggle is disabled. When this toggle is enabled, the toggle "Add handling costs" automatically switched to "no" and becomes disabled as all the input fields for "ranges".
+- **Billing.** 2 Checkboxes are present for this section: "According to total price.""and "According to total weight.". 
+- **Tax.** this is a drop-down menu, shows all the possible tax rates depending on the countries installed on the shop.
+- **Out-of-range behavior** this is a drop-down menu with the possible options: "Apply the cost of the highest defined range" and "Disable carrier".
+A tooltip is displayed: "Out-of-range behavior occurs when no defined range matches the customer's cart (e.g. when the weight of the cart is greater than the highest weight limit is defined by the weight ranges)."
+
```

Comment:
If the total weight or total price of the cart is not in one of the defined ranges and if the chosen option is "Disable carrier", then the carrier will be disabled.
If the total weight or total price of the cart is not in one of the defined ranges and if the chosen option is "Apply the cost of the highest defined range", then the carrier price will be the one corresponding to the highest range.